pages/content/amp-dev/page-experience.html

```diff
@@ -48,7 +48,23 @@ <h1 class="ap-o-stage-content-headline">
       to delivering a great user experience on the web.
     </p>
   </section>
+
   {% do doc.styles.addCssFile('css/components/molecules/input-bar.css') %}
+  <amp-state id="pixi">
```

Comment:
nit: can we call this `i18n`? Are there other aspects we plan to use `amp-state` for?

Reply:
I think it's safer to namespace pixi specific amp-state to avoid conflicts with general amp.dev functionality in the future.